nodejs/clients.js

```diff
@@ -1,23 +1,22 @@
-const { readMasterKey, ClientBuilder } = require("./helpers")
+const { readMasterKey, CsfleHelper } = require("./helpers")
 
 const localMasterKey = readMasterKey()
 
-const clientBuilder = new ClientBuilder({
+const csfleHelper = new CsfleHelper({
```

Comment:
I'm a little concerned about jamming "all the scary stuff" into a class called `CsfleHelper`. Do you think it's too cumbersome for the user? I wonder if that should be fed back to the FLE advisory group. IMO, it's a little bit cumbersome to setup, but in this example where we are showing off exactly how to configure CSFLE, we should be doing it step-by-step in a single file so there's no need to jump around to know what's going on.

Reply:
We did this because we didn't want a single monolithic file. This code is meant to accompany the csfle guide we have [here](https://docs.mongodb.com/ecosystem/use-cases/client-side-field-level-encryption-guide/) which explains everything in detail and is really meant to satiate more advanced engineers like those we user tested who "just want a contained runnable example at first".